Release `0.1.3`

robbot006 opened the pull request:
This is a draft of changelog for the `0.1.3` release.
Here you can find all related translations changes: [0.1.2...master](https://github.com/flarum-lang/uzbek/compare/0.1.2...master).

> ### ⚠️ Do not merge this pull request manually ⚠️
> 
> You should [approve](https://github.com/rob006-software/flarum-translations/wiki/How-to-approve-release-pull-request) this pull request instead. After approving, I will automatically update it (release date needs to be adjusted), merge, and tag a new release.

<details>
<summary>Show release notes preview</summary>

## 0.1.3

<!-- release-notes-begin -->
**General changes**:

* Updated Day.js translations.
* Updated Flarum core translations.
* Updated validation translations.


**Added support for new extensions**:

* [`flarum/mentions`](https://github.com/flarum/mentions)
* [`fof/best-answer`](https://github.com/FriendsOfFlarum/best-answer)
* [`fof/custom-footer`](https://github.com/FriendsOfFlarum/custom-footer)
* [`fof/discussion-language`](https://github.com/FriendsOfFlarum/discussion-language)
* [`fof/formatting`](https://github.com/FriendsOfFlarum/formatting)
* [`fof/forum-statistics-widget`](https://github.com/FriendsOfFlarum/forum-statistics-widget)
* [`fof/gamification`](https://github.com/FriendsOfFlarum/gamification)
* [`fof/links`](https://github.com/FriendsOfFlarum/links)
* [`fof/oauth`](https://github.com/FriendsOfFlarum/oauth)
* [`fof/pages`](https://github.com/FriendsOfFlarum/pages)
* [`fof/polls`](https://github.com/FriendsOfFlarum/polls)
* [`fof/recaptcha`](https://github.com/FriendsOfFlarum/recaptcha)
* [`fof/share-social`](https://github.com/FriendsOfFlarum/share-social)
* [`justoverclock/flarum-ext-infocards`](https://github.com/justoverclockl/flarum-ext-infocards)
* [`justoverclock/flarum-ext-welcomebox`](https://github.com/justoverclockl/flarum-ext-welcomebox)
* [`malago/flarum-achievements`](https://github.com/malago86/flarum-achievements)
* [`nearata/flarum-ext-tags-color-generator`](https://github.com/Nearata/flarum-ext-tags-color-generator)
* [`nyu8/flarum-email-filter`](https://github.com/nyu8/flarum-email-filter)
* [`v17development/flarum-support`](https://extiverse.com/extension/v17development/flarum-support)


**Updated translations for extensions**:

* [`acpl/mobile-tab`](https://github.com/android-com-pl/mobile-tab)
* [`askvortsov/flarum-rich-text`](https://github.com/askvortsov1/flarum-rich-text)
* [`blomstra/payments`](https://extiverse.com/extension/blomstra/payments)
* [`clarkwinkelmann/flarum-ext-carving-contest`](https://github.com/clarkwinkelmann/flarum-ext-carving-contest)
* [`datitisev/flarum-backup`](https://extiverse.com/extension/datitisev/flarum-backup)
* [`davwheat/flarum-ext-ads`](https://github.com/davwheat/flarum-ext-ads)
* [`flarum/likes`](https://github.com/flarum/likes)
* [`flarum/lock`](https://github.com/flarum/lock)
* [`kilowhat/flarum-ext-formulaire`](https://extiverse.com/extension/kilowhat/flarum-ext-formulaire)


All changes: [0.1.2...0.1.3](https://github.com/flarum-lang/uzbek/compare/0.1.2...0.1.3).



<!-- release-notes-end -->

</details>

rob006 commented:
@JeongJun-Lee Just as a note, this is an automated PR generated by bot. The idea is to simplify release process for language pack maintainers - all you need is to approve this pull request and bot will handle the rest. You can read more about the process at forum: https://discuss.flarum.org/d/20807-simplify-translation-process-with-weblate/76 . If you don't want to use this feature, let me know.

robbot006 commented:
Success! Now you can announce new release on [forum](https://discuss.flarum.org/t/languages):

~~~markdown
## Version [`0.1.3`](https://github.com/flarum-lang/uzbek/releases/tag/0.1.3)

**General changes**:

* Updated Day.js translations.
* Updated Flarum core translations.
* Updated validation translations.


**Added support for new extensions**:

* [`flarum/mentions`](https://github.com/flarum/mentions)
* [`fof/best-answer`](https://github.com/FriendsOfFlarum/best-answer)
* [`fof/custom-footer`](https://github.com/FriendsOfFlarum/custom-footer)
* [`fof/discussion-language`](https://github.com/FriendsOfFlarum/discussion-language)
* [`fof/formatting`](https://github.com/FriendsOfFlarum/formatting)
* [`fof/forum-statistics-widget`](https://github.com/FriendsOfFlarum/forum-statistics-widget)
* [`fof/gamification`](https://github.com/FriendsOfFlarum/gamification)
* [`fof/links`](https://github.com/FriendsOfFlarum/links)
* [`fof/oauth`](https://github.com/FriendsOfFlarum/oauth)
* [`fof/pages`](https://github.com/FriendsOfFlarum/pages)
* [`fof/polls`](https://github.com/FriendsOfFlarum/polls)
* [`fof/recaptcha`](https://github.com/FriendsOfFlarum/recaptcha)
* [`fof/share-social`](https://github.com/FriendsOfFlarum/share-social)
* [`justoverclock/flarum-ext-infocards`](https://github.com/justoverclockl/flarum-ext-infocards)
* [`justoverclock/flarum-ext-welcomebox`](https://github.com/justoverclockl/flarum-ext-welcomebox)
* [`malago/flarum-achievements`](https://github.com/malago86/flarum-achievements)
* [`nearata/flarum-ext-tags-color-generator`](https://github.com/Nearata/flarum-ext-tags-color-generator)
* [`nyu8/flarum-email-filter`](https://github.com/nyu8/flarum-email-filter)
* [`v17development/flarum-support`](https://extiverse.com/extension/v17development/flarum-support)


**Updated translations for extensions**:

* [`acpl/mobile-tab`](https://github.com/android-com-pl/mobile-tab)
* [`askvortsov/flarum-rich-text`](https://github.com/askvortsov1/flarum-rich-text)
* [`blomstra/payments`](https://extiverse.com/extension/blomstra/payments)
* [`clarkwinkelmann/flarum-ext-carving-contest`](https://github.com/clarkwinkelmann/flarum-ext-carving-contest)
* [`datitisev/flarum-backup`](https://extiverse.com/extension/datitisev/flarum-backup)
* [`davwheat/flarum-ext-ads`](https://github.com/davwheat/flarum-ext-ads)
* [`flarum/likes`](https://github.com/flarum/likes)
* [`flarum/lock`](https://github.com/flarum/lock)
* [`kilowhat/flarum-ext-formulaire`](https://extiverse.com/extension/kilowhat/flarum-ext-formulaire)


All changes: [0.1.2...0.1.3](https://github.com/flarum-lang/uzbek/compare/0.1.2...0.1.3).

In order to update run these commands:

```console
composer update flarum-lang/uzbek
php flarum cache:clear
```

~~~
